src/standalone/Vegetation/pfts.jl

```diff
@@ -0,0 +1,534 @@
+    1467–1496, https://doi.org/10.5194/gmd-11-1467-2018, 2018.
+    abb. as Bonan et al. 2018
+"""
+
+export Pft, default_pfts, params_from_pfts, pft_param_list
+
+using DataFrames
+
+# List of parameters that PFTs need to define in order to be valid
+pft_param_list = [
+    # TwoStreamModel parameters
+    :Ω,          # Clumping Index
+    :α_PAR_leaf, # PAR Leaf Reflectance
+    :α_NIR_leaf, # Near-infrared Leaf Reflectance
+    :τ_PAR_leaf, # PAR Leaf Transmittance
+    :τ_NIR_leaf, # Near-infrared Leaf Transmittance
+    :ϵ_canopy,   # Canopy Emissivity
+    :χl,         # Leaf/Stem orientation Index
+    :ac_canopy,  # Canopy Specific Heat per Area
+
+    # Conductance Model
+    :g1, # Medlyn CO2 Sensitivity
+
+    # Photosynthesis model
+    :Vcmax25, # Max rate of carboxylation of Rubisco
+    # When running optimality Farquhar model, this param is not used
+
+    # Plant Hydraulics and general plant parameters
+    :f_root_to_shoot, # Root to Shoot Ratio
+    :K_sat_plant,     # Hydraulic Conductivity at Saturation
+    :ψ63,             # Xylem water potential at 63% loss of conductivity
+    :plant_ν,         # Plant porosity
+    :rooting_depth,   # Plant Rooting Depth
+]
+
+
+"""
+Define a PFT type that can be used to store the parameters for each PFT.
+Each PFT has a name and a list of parameters. The parameters are stored as 
+NamedTuples mapping parameter names to values. The inner constructor checks to
+ensure all required parameters are defined for the PFT.
+"""
+struct Pft
+    name::String
+    parameters::NamedTuple
+
+    # Inner constructor to verify all required parameters are defined
+    function Pft(name, parameters)
+        defined_param_names = keys(parameters)
+        for param in pft_param_list
+            if param ∉ defined_param_names
+                error("PFT $name is missing parameter $param")
+            end
+        end
+        new(name, parameters)
+    end
+end
+
+
+NET_Temp = Pft(
+    "Temperate Needleleaf Evergreen Tree",
+    (
+        # Radiative Transfer Model
+        Ω = 0.74,          # Unitless ⋅ Chen et al. 2012 Table 1
+        α_PAR_leaf = 0.07, # Unitless ⋅ CLM5.0 Table 2.3.1
+        α_NIR_leaf = 0.35, # Unitless ⋅ CLM5.0 Table 2.3.1
+        τ_PAR_leaf = 0.05, # Unitless ⋅ CLM5.0 Table 2.3.1
+        τ_NIR_leaf = 0.10, # Unitless ⋅ CLM5.0 Table 2.3.1
+        ϵ_canopy = 0.982,  # Unitless ⋅ Ma et al. 2019
+        χl = 0.01,         # Unitless ⋅ CLM2 PFT Data
+        ac_canopy = 2234,  # Jm^-2K^-1 ⋅ Bonan et al. 2018
+
+        # Stomatal Conductance Model
+        g1 = 74.31, # kPa^(1/2) ⋅ CLM5.0 Table 2.9.1
+
+        # Photosynthesis Model
+        Vcmax25 = 5.1e-5, # mol CO2/m^2/s ⋅ CLM2 PFT Data
+
+        # Plant Hydraulics and general plant parameters
+        f_root_to_shoot = 1.0, # Unitless ⋅ CLM2 PFT Data
+        K_sat_plant = 5e-9,    # m/s
+        ψ63 = -4 / 0.0098,     # / MPa to m
+        plant_ν = 2e-4,        # Unitless
```

Comment:
maybe just double check the order of magnitude on this given the /1000 vs *1000 typo above?
fine to set these all equal for now :)